LDAP: Add `skip_org_role_sync` configuration option (#56679)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/ldap.md

In order to use LDAP integration you'll first need to enable LDAP in the [main config file]({{< relref "../../configure-grafana/" >}}) as well as specify the path to the LDAP
specific configuration file (default: `/etc/grafana/ldap.toml`).

```ini
[auth.ldap]
# Set to `true` to enable LDAP integration (default: `false`)
enabled = true

> # Path to the LDAP specific configuration file (default: `/etc/grafana/ldap.toml`)

Reviewer: ```suggestion
# The path to the LDAP specific configuration file (default: `/etc/grafana/ldap.toml`)
```

config_file = /etc/grafana/ldap.toml

> # Allow sign up should almost always be true (default) to allow new Grafana users to be created (if LDAP authentication is ok). If set to

Reviewer: ```suggestion
# Allow sign-up should almost always be true (default) which enables new Grafana users to be created when LDAP authentication is configured). If set to
```

> # false only pre-existing Grafana users will be able to login (if LDAP authentication is ok).

Reviewer: ```suggestion
# `false`, only existing Grafana users can sign in when LDAP authentication is configured.
```

allow_sign_up = true
```

## Disable org role synchronization

> If you are using LDAP to authenticate users but don't use role mapping and prefer manually assigning organizations

Reviewer: ```suggestion
If you use LDAP to authenticate users but don't use role mapping, and prefer to manually assign organizations
```

> # Allow sign up should almost always be true (default) to allow new Grafana users to be created (if LDAP authentication is ok). If set to

Reviewer: ```suggestion
# Allow sign-up should almost always be true (default) which enables new Grafana users to be created when LDAP authentication is configured. If set to
```
